README.rst

```diff
@@ -1,97 +1,118 @@
 
-Once ``tahoe --version`` works, see `<docs/running.rst>`_ to learn how to set
-up your first Tahoe-LAFS node.
+Please read more about Tahoe-LAFS architecture `here <docs/architecture.rst>`__.
 
-LICENCE
-=======
+✅ Installation
+---------------
 
-Copyright 2006-2018 The Tahoe-LAFS Software Foundation
+For more detailed instructions, read `docs/INSTALL.rst <docs/INSTALL.rst>`__ .
 
-You may use this package under the GNU General Public License, version 2 or,
-at your option, any later version. You may use this package under the
-Transitive Grace Period Public Licence, version 1.0, or at your option, any
-later version. (You may choose to use this package under the terms of either
-licence, at your option.) See the file `COPYING.GPL`_ for the terms of the
-GNU General Public License, version 2. See the file `COPYING.TGPPL`_ for
-the terms of the Transitive Grace Period Public Licence, version 1.0.
+-  `Building Tahoe-LAFS on Windows <docs/windows.rst>`__
 
-See `TGPPL.PDF`_ for why the TGPPL exists, graphically illustrated on three
-slides.
+-  `OS-X Packaging <docs/OS-X.rst>`__
 
-.. _OSPackages: https://tahoe-lafs.org/trac/tahoe-lafs/wiki/OSPackages
-.. _Mac: docs/OS-X.rst
-.. _pip: https://pip.pypa.io/en/stable/installing/
-.. _COPYING.GPL: https://github.com/tahoe-lafs/tahoe-lafs/blob/master/COPYING.GPL
-.. _COPYING.TGPPL: https://github.com/tahoe-lafs/tahoe-lafs/blob/master/COPYING.TGPPL.rst
-.. _TGPPL.PDF: https://tahoe-lafs.org/~zooko/tgppl.pdf
+Once tahoe --version works, see `docs/running.rst <docs/running.rst>`__ to learn how to set up your first Tahoe-LAFS node.
 
-----
+💬 Community
+------------
 
+Get involved with the Tahoe-LAFS community:
+
+-  Chat with Tahoe-LAFS developers at #tahoe-lafs chat on irc.freenode.net or `Slack <https://join.slack.com/t/tahoe-lafs/shared_invite/zt-jqfj12r5-ZZ5z3RvHnubKVADpP~JINQ>`__.
+
+-  Join our `weekly conference calls <https://www.tahoe-lafs.org/trac/tahoe-lafs/wiki/WeeklyMeeting>`__ with core developers and interested community members.
+
+-  Subscribe to `the tahoe-dev mailing list <https://www.tahoe-lafs.org/cgi-bin/mailman/listinfo/tahoe-dev>`__, the community forum for discussion of Tahoe-LAFS design, implementation, and usage.
+
+🤗 Contributing
+---------------
+
+As a community-driven open source project, Tahoe-LAFS welcomes contributions of any form:
+
+-  `Code patches <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Patches>`__
+
+-  `Documentation improvements <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Doc>`__
+
+-  `Bug reports <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/HowToReportABug>`__
+
+-  `Patch reviews <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/PatchReviewProcess>`__
+
+Before authoring or reviewing a patch, please familiarize yourself with the `Coding Standard <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/CodingStandards>`__ and the `Contributor Code of Conduct <https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/CODE_OF_CONDUCT.md>`__.
```

Comment:
Does the `CODE_OF_CONDUCT.md` link work as `CODE_OF_CONDUCT <docs/CODE_OF_CONDUCT.md>` like the other relative links?  If so that seems preferable.